test/integration/framework/master_utils.go

```diff
@@ -214,15 +214,15 @@ func startMasterOrDie(masterConfig *master.Config, incomingServer *httptest.Serv
 		Groups: []string{user.SystemPrivilegedGroup},
 	}
 
-	tokenAuthenticator := authenticator.NewAuthenticatorFromTokens(tokens)
+	tokenAuthenticator := authenticatorfactory.NewAuthenticatorFromTokens(tokens)
```

Comment:
> NewFromTokens

Want it changed during the move or can I convince you that we start a followup issue tracking renames and package moves for cleaning up after we carve it all up?